package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-native-file-viewer",
+  "name": "@qmatteoq/react-native-file-viewer",
```

Comment:
Probably not intended to have the `qmatteoq` part?

Reply:
Correct, thanks for the sanity check :-) It was part of my tests, in which I've tried to publish my work as a NPM package to test that the whole installation process from NPM was working fine.

Reply:
I haven't used it myself yet, but I hear [Verdaccio](https://verdaccio.org/) helps for testing NPM package changes like this.

Reply:
this hasn't been fixed yet